scripts/funcotator/data_sources/gnomAD/liftoverVcf.wdl

```diff
@@ -0,0 +1,373 @@
+
+        endTime=`date +%s.%N`
+        echo "EndTime: $endTime" >> timingInformation.txt
+        elapsedTime=`echo "scale=5;$endTime - $startTime" | bc`
+        echo "Elapsed Time: $elapsedTime" >> timingInformation.txt
+     >>>
+
+     # ------------------------------------------------
+     # Runtime settings:
+     runtime {
+         docker: gatk_docker
+         memory: machine_mem + " MB"
+         disks: "local-disk " + select_first([disk_space_gb, default_disk_space_gb]) + if use_ssd then " SSD" else " HDD"
+         bootDiskSizeGb: select_first([boot_disk_size_gb, default_boot_disk_size_gb])
+         preemptible: 0
+         cpu: select_first([cpu, 1])
+     }
+
+     # ------------------------------------------------
+     # Outputs:
+     output {
+         File lifted_over_vcf         = "${lifted_over_vcf_name}"
+         File lifted_over_rejects_vcf = "${lifted_over_rejects_vcf_name}"
+     }
+ }
+
+task MergeVcfsTask {
+     # inputs
+     Array[File] input_vcfs
+
+     # outputs
+     String output_vcf_file
+
+     # runtime
+     String gatk_docker
+     File? gatk_override
+     Int? mem
+     Int? preemptible_attempts
+     Int? disk_space_gb
+     Int? cpu
+     Int? boot_disk_size_gb
+
+     Boolean use_ssd = false
+
+     # You may have to change the following two parameter values depending on the task requirements
+     Int default_ram_mb = 3000
+     # WARNING: In the workflow, you should calculate the disk space as an input to this task (disk_space_gb).  Please see [TODO: Link from Jose] for examples.
+     Int default_disk_space_gb = 100
+
+     Int default_boot_disk_size_gb = 15
+
+     # Mem is in units of GB but our command and memory runtime values are in MB
+     Int machine_mem = if defined(mem) then mem *1000 else default_ram_mb
+     Int command_mem = machine_mem - 1000
+
+     command <<<
+         set -e
+         export GATK_LOCAL_JAR=${default="/root/gatk.jar" gatk_override}
+
+        startTime=`date +%s.%N`
+        echo "StartTime: $startTime" > timingInformation.txt
+
+         gatk --java-options "-Xmx${command_mem}m -DGATK_STACKTRACE_ON_USER_EXCEPTION=true" \
+            MergeVcfs \
+             -I ${sep=' -I ' input_vcfs} \
+             -O ${output_vcf_file}
+
+        endTime=`date +%s.%N`
+        echo "EndTime: $endTime" >> timingInformation.txt
+        elapsedTime=`echo "scale=5;$endTime - $startTime" | bc`
+        echo "Elapsed Time: $elapsedTime" >> timingInformation.txt
+     >>>
+
+     runtime {
+         docker: gatk_docker
+         memory: machine_mem + " MB"
+         disks: "local-disk " + select_first([disk_space_gb, default_disk_space_gb]) + if use_ssd then " SSD" else " HDD"
+         bootDiskSizeGb: select_first([boot_disk_size_gb, default_boot_disk_size_gb])
+         preemptible: 0 
+         cpu: select_first([cpu, 1])
+     }
+
+     output {
+         File vcf_file = "${output_vcf_file}"
+     }
+ }
+
+
+task IndexFeatureFileTask {
```

Comment:
You may want to consider subworkflows (or WDL imports), since I think you have repeated tasks.  If you would like to file an issue for that/punt, I am okay, for now.

Reply:
I'm fine with subworkflows/imports.  I just had not done it before so I wanted to make something I knew would work.